src/OpenTelemetry.Extensions.PersistentStorage/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 1.0.0-alpha.3
 
+* [Update implementation to use
+  Opentelemetry.Extensions.PersistentStorage.Abstractions](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/363)
```

Comment:
This should be under "Unreleased" section.

Reply:
Done